firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,6 @@
 # Unreleased
+- [fixed] Fixed an issue where query results were temporarily missing documents
```

Comment:
This is actually going to go out with 20.2.0 if we submit today, so please move down one section.

Reply:
Also, please add a reference to the issue `(#155)` before the trailing period.

Reply:
Done.